include/evmc/evmc.h

```diff
@@ -685,6 +685,21 @@ typedef struct evmc_result (*evmc_execute_fn)(struct evmc_instance* instance,
                                               size_t code_size);
 
 
+struct evmc_tracer_context;
+
+typedef void (*evmc_trace_callback)(struct evmc_tracer_context* context,
+                                    int depth,
+                                    int step,
+                                    size_t code_offset,
```

Comment:
The problem with "pc" name I have is not everyone knows exactly that it is exactly. I think `code_offest` === `pc`.

Reply:
So what is `code_offset` and what is `step` ?

Reply:
The `step` is the number of executed instructions so far (only increases).
The `code_offset` is the position in the code where we are currently. It's different than the `step` because of jump and push instructions.

Reply:
`execution_step` perhaps? Not sure, `step`, without any documentation gave me the wrong indication about what it is.

Why don't you want to have a documentation above it?

Reply:
We can also remove this because this can be implemented easily on the client side.

Reply:
I will add documentation soon. I just need to finish other PRs first.